tokio-util/src/net/unix/mod.rs

```diff
@@ -0,0 +1,76 @@
+//! Unix domain socket helpers.
+
+use std::future::Future;
+use std::io::Result;
+use std::pin::Pin;
+
+/// Listening address.
+#[derive(Debug)]
+pub enum ListenAddr {
+    /// Socket address.
+    SocketAddr(std::net::SocketAddr),
+    /// Unix socket.
+    UnixSocket(tokio::net::unix::SocketAddr),
+}
```

Comment:
This should probably be an associated type of the `Listener` trait like how `Io` is set up.

Reply:
Done in 9bda024